tests/sentry/api/endpoints/test_group_notes.py

```diff
@@ -52,3 +52,24 @@ def test_simple(self):
             'text': 'hello world',
         })
         assert response.status_code == 400, response.content
+
+    def test_with_mentions(self):
+        group = self.group
+
+        self.login_as(user=self.user)
+
+        url = '/api/0/issues/{}/comments/'.format(group.id)
+
+        response = self.client.post(url, format='json', data={
+            'text': '**meredith@getsentry.com** is fun',
+            'mentions': [u'8']
+        })
+        assert response.status_code == 400, response.content
+
+        user_id = six.text_type(self.user.id)
+
+        response = self.client.post(url, format='json', data={
+            'text': '**meredith@getsentry.com** is so fun',
+            'mentions': [u'%s' % user_id]
+        })
+        assert response.status_code == 201, response.content
```

Comment:
can you also add more to these tests (and possibly add comments as to what each are testing)? one thing that you should definitely test is try to mention an existing user who is not part of the organization and make sure you get the appropriate validation error.